File: Python/SingleLinkedList.py
class Node:    
	def __init__(self,data):
		self.data = data;
		self.next = None;


class Linked:

    def printList(self):
        temp = self.first
        while temp:
            print temp.data
            temp = temp.next


	def __init__(self):
		self.first = None


    

if __name__ == '__main__':
    list = Linked()
    
    list.first = Node(10)
    second = Node(20)
    third = Node(30)
    list.first.next = second
    second.next = third

    print list.first.data
    print list.first.next.data
    print list.first.next.next.data
    
    list.printList()

File: C/SingleLinkedList.c
#include <stdio.h>
#include <stdlib.h>

struct node{
	struct node *next;
	int data;
};

void printList(struct node *head){
	while(head != NULL){
		printf("%d\n",head->data);
		head = head->next;
	}
}


int main(){
	struct node  *first,*second,*third;

	first = (struct node *)malloc(sizeof(struct node));

	first->data = 10;
	second = (struct node *)malloc(sizeof(struct node));

	second->data = 20;
	first->next = second;

	third = (struct node *)malloc(sizeof(struct node));

	third->data = 30;
	second->next = third;

	printf("%d\n",first->data); //10
	printf("%d\n",first->next->data); //20
	printf("%d\n",first->next->next->data); //30


	printList(first);
	return 0;
}